### PR TITLE
Gen 1956   moved dependencies

### PR DESCRIPTION
**Pull request by ngprnk**
# Change log 

1. Moved packages from dependencies to dev dependencies so that they are not downloaded in production. 
2. Removed the hotjar stuff

### CODE REVIEW

**Comment by conorpirie** [comment]
merge conflict

**Comment by conorpirie** [comment]
needs more detail on what it does and how to use it. right now it is disabled by default, how is it used?

File: package.json [hunk 1]
```diff
@@ -121,12 +120,9 @@
     "react-table": "6.8.6",
     "redux": "^4.0.0",
     "redux-batched-subscribe": "^0.1.6",
-    "redux-devtools-extension": "^2.13.5",
     "redux-logger": "^3.0.6",
     "redux-observable": "^0.18.0",
     "rxjs": "^5.5.10",
-    "slate": "0.40.2",
```
Comment (conorpirie) [inline package.json]: Module not found: Error: Can't resolve 'slate' in '/Users/conorpirie/projects/genny/alyson-v7/node_modules/slate-base64-serializer/lib'

Comment (conorpirie) [reply package.json]: Module not found: Error: Can't resolve 'slate-html-serializer' in '/Users/conorpirie/projects/genny/alyson-v7/src/views/components-legacy/input/rich-text-editor'

Comment (ngprnk) [reply package.json]: #24 will fix this issue, that can be merged first and then this. I have got rid of these packages on that PR.

**Pull request merged**